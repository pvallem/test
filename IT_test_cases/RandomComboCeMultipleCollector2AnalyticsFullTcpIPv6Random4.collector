bidir_flow_info {
  key {
    src_address: "/q@+\256\346^\216}\217\253DB\266\257\326"
    dst_address: "fv\315\3745\322\307\334{+\254U=4J\270"
    src_port: 20360
    dst_port: 25580
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV6
  }
  reverse_flow_info {
    start_time: 1388563200000010
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000010
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\002\002\002\002\002\001"
    dst_address: "\002\002\002\002\002d"
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: MAC
    ether_type: 34827
  }
  flow_info {
    start_time: 1388563200000020
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000020
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\002\002\002\002\002\000"
    dst_address: "\002\002\002\002\002d"
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: MAC
    ether_type: 34827
  }
  flow_info {
    start_time: 1388563200000000
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 2
    }
    end_time: 1388563200000040
    sensor_id: "127.0.0.1:5675"
  }
}
bidir_flow_info {
  key {
    src_address: "\007H\033\210\325 \333\021\206\312\321\031\354\374\3373"
    dst_address: "\365;W<\033\317]TQBW\010\262\305\351J"
    src_port: 19432
    dst_port: 39856
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV6
  }
  flow_info {
    start_time: 1388563200000030
    ip_info {
      qos: 0
    }
    flow_features {
      packet_count: 1
    }
    end_time: 1388563200000030
    sensor_id: "127.0.0.1:5675"
  }
}
