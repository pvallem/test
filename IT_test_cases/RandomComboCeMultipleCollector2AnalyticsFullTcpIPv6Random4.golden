flow_info {
  key {
    src_address: "\002\002\002\002\002\000"
    dst_address: "\002\002\002\002\002d"
    tenant_id: 10
    flow_start_time: 1388563200000000
    key_type: MAC
    ether_type: 34827
  }
  start_time: 1388563200000000
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 2
  }
  end_time: 1388563200000040
}
flow_info {
  key {
    src_address: "fv\315\3745\322\307\334{+\254U=4J\270"
    dst_address: "/q@+\256\346^\216}\217\253DB\266\257\326"
    src_port: 25580
    dst_port: 20360
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000010
    key_type: IPV6
  }
  start_time: 1388563200000010
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000010
}
flow_info {
  key {
    src_address: "\002\002\002\002\002\001"
    dst_address: "\002\002\002\002\002d"
    tenant_id: 10
    flow_start_time: 1388563200000020
    key_type: MAC
    ether_type: 34827
  }
  start_time: 1388563200000020
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000020
}
flow_info {
  key {
    src_address: "\007H\033\210\325 \333\021\206\312\321\031\354\374\3373"
    dst_address: "\365;W<\033\317]TQBW\010\262\305\351J"
    src_port: 19432
    dst_port: 39856
    proto: 6
    tenant_id: 10
    flow_start_time: 1388563200000030
    key_type: IPV6
  }
  start_time: 1388563200000030
  ip_info {
    qos: 0
  }
  flow_features {
    packet_count: 1
  }
  end_time: 1388563200000030
}
